Formula/s/synergy-core.rb

```diff
@@ -1,8 +1,9 @@
 class SynergyCore < Formula
   desc "Synergy, the keyboard and mouse sharing tool"
   homepage "https://symless.com/synergy"
-  url "https://github.com/symless/synergy-core/archive/refs/tags/1.14.6.19-stable.tar.gz"
-  sha256 "01854ec932845975cd81363bed8276b95c07a607050683fc1b74b7126199de79"
+  url "https://github.com/symless/synergy/archive/refs/tags/1.15.1+r1.tar.gz"
+  version "1.15.1"
```

Comment:
Isn't the actual version `1.15.1+r1`?

Reply:
I have added the following explanation to the commit message:

    The filename for this new release of synergy-core is "1.15.1+r1.tar.gz",
    which may suggest that the actual release version of "1.15.1+r1" instead
    of "1.15.1". However, the GitHub Releases page for this release is
    titled "1.15.1":
    
      https://github.com/symless/synergy/releases/tag/1.15.1%2Br1
    
    Furthermore, the command line tools built by the formula all identify
    themselves as version 1.15.1:
    
      $ synergy --version | head -n 1
      [2024-09-06T21:15:24] INFO: Synergy v1.15.1
    
      $ synergyc --version | head -n 1
      synergyc v1.15.1, protocol v1.8
    
      $ synergys --version | head -n 1
      synergys v1.15.1, protocol v1.8
    
    Finally, the GUI client built by the formula also identifies itself
    as version 1.15.1:
    
      https://gist.githubusercontent.com/cathyjf/05da9ec8f98d344a3673835b4c58d53a/raw/5f8f659d885568eb768f0cebd0eb83da1250f494/Screenshot%25202024-09-06%2520at%25209.17.38%25E2%2580%25AFPM.png
    
    For these reasons, I have concluded that the version of the release
    is "1.15.1", rather than "1.15.1+r1".

If you still want me to change it to "1.15.1+r1", let me know.

Reply:
No, that's fine, thanks!